OrbitGl/ModulesDataView.cpp

```diff
@@ -196,6 +196,11 @@ void ModulesDataView::SetModules(int32_t process_id,
   OnDataChanged();
 }
 
+//-----------------------------------------------------------------------------
```

Comment:
nit: drop this

Reply:
removed it in quite a few places that I touched; done